sdk/core/System.ClientModel/src/Pipeline/ClientRetryPolicy.cs

```diff
@@ -175,20 +242,63 @@ protected virtual bool ShouldRetry(PipelineMessage message, Exception? exception
         return isRetriable;
     }
 
+    /// <summary>
+    /// A method that can be overridden by derived types to customize the default
+    /// <see cref="ClientRetryPolicy"/> logic. It is called from
+    /// <see cref="Process(PipelineMessage, IReadOnlyList{PipelinePolicy}, int)"/>
+    /// after control has been returned from the policy at the position
+    /// after the retry policy in the pipeline.
+    /// </summary>
+    /// <param name="message">The <see cref="PipelineMessage"/> for this
+    /// pipeline invocation.</param>
+    /// <param name="exception">The exception, if any, that was thrown from
+    /// a policy after the retry policy in the pipeline.</param>
     protected virtual ValueTask<bool> ShouldRetryAsync(PipelineMessage message, Exception? exception)
         => new(ShouldRetry(message, exception));
 
+    /// <summary>
+    /// A method that can be overridden by derived types to customize the default
+    /// <see cref="ClientRetryPolicy"/> logic. It is called from
+    /// <see cref="Process(PipelineMessage, IReadOnlyList{PipelinePolicy}, int)"/>
+    /// to determine how long the policy should wait before re-sending the request.
+    /// </summary>
+    /// <param name="message">The <see cref="PipelineMessage"/> for this
+    /// pipeline invocation.</param>
+    /// <param name="tryCount">A number indicating how many times the policy has
+    /// tried to send the request.</param>
+    /// <returns></returns>
     protected virtual TimeSpan GetNextDelay(PipelineMessage message, int tryCount)
     {
         // Default implementation is exponential backoff
         return TimeSpan.FromMilliseconds((1 << (tryCount - 1)) * _initialDelay.TotalMilliseconds);
     }
 
+    /// <summary>
+    /// A method that can be overridden by derived types to customize the default
+    /// <see cref="ClientRetryPolicy"/> logic. It is called from
+    /// <see cref="Process(PipelineMessage, IReadOnlyList{PipelinePolicy}, int)"/>
+    /// to wait the time interval returned by <see cref="GetNextDelay(PipelineMessage, int)"/>.
+    /// </summary>
+    /// <param name="time">The amount of time to wait before trying to send the
+    /// request again.</param>
+    /// <param name="cancellationToken">The <see cref="CancellationToken"/> used
+    /// to cancel the wait if needed.</param>
+    /// <returns></returns>
```

Comment:
```suggestion
```

There's no value, just a task.  I tend to omit the `<return>` tag for these rather than saying something like "A task representing the operation", but I'm good either way you choose.